### PR TITLE
Update FAQ.md

### PR DESCRIPTION
**Pull request by TheVillageGuy**
added a line break to avoid confusion and small grammar fix

### CODE REVIEW

**Comment by TheVillageGuy** [comment]
another change needs to made, what's the best way to force this?

( 
sudo /usr/sbin/rabbitmq-plugins enable rabbitmq_management
)
is the line it should be to work

File: FAQ.md [hunk 1]
```diff
@@ -80,7 +80,8 @@ Execute: `sudo rabbitmqctl status`, that should provide details about your Rabbi
 
 Ensure you do not see any connection errors in your `var/log/prod.log` file.
 
-Talking about plugins, we advise to also enable the `rabbitmq_management` plugin by execute: `rabbitmq-plugins enable rabbitmq_management`.
+Talking about plugins, we advise to also enable the `rabbitmq_management` plugin by executing: 
+`rabbitmq-plugins enable rabbitmq_management`.
```
Comment (TheVillageGuy) [inline FAQ.md:84]: sudo /usr/sbin/rabbitmq-plugins enable rabbitmq_management

Comment (melroy89) [reply FAQ.md]: Fixed.

**Comment by melroy89** [comment]
`/usr/sbin/` is not needed, that is part of the default path of the root user.

**Comment by TheVillageGuy** [comment]
> `/usr/sbin/` is not needed, that is part of the default path of the root user.

You assume I run it as root, but I don't

**Comment by melroy89** [comment]
> > `/usr/sbin/` is not needed, that is part of the default path of the root user.
> 
> You assume I run it as root, but I don't. It's not in my path.

I mean you run via `sudo` right? It should be part of that.

**Comment by melroy89** [comment]
Check your `/etc/sudoers` file:

```
Defaults        secure_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin"
```

**Comment by TheVillageGuy** [comment]
You are right, I was blind

**Review by melroy89 — APPROVED** [review]

**Pull request merged**